modules/QnA/resources/qnaService.js

```diff
@@ -584,12 +584,12 @@ DAL for Q&A Module
                     request['filter:updatedAtGreaterThanOrEqual'] = lastUpdatedAt;
                 }
 
-                // build list code cue point request
+                // build list code cue point request for qna settings (checking both new and old tags for BC)
                 var codeCuePointListRequest = {
                     'service': 'cuepoint_cuepoint',
                     'action': 'list',
                     'filter:entryIdEqual': entryId,
-                    'filter:tagsLike':'WEBCASTSTATETAG',
+                    'filter:tagsMultiLikeOr':'player-qna-settings-update,WEBCASTSTATETAG',
```

Comment:
we removed ** WEBCASTSTATETAG** tag and will not send it anymore, you can remove it. If you are saw other places that still use it let me know